it/utils/AppealWiremock.scala

```diff
@@ -26,6 +26,9 @@ trait AppealWiremock {
       .willReturn(
         aResponse()
           .withStatus(status)
+          .withHeader("Authorization" ,"pega-bearerToken")
+          .withHeader("CorrelationId", "1111111-qqqq-2222-eeee")
+          .withHeader("Environment", "pegaEnvironment")
```

Comment:
Up to you whether you leave this in but the response itself probably won't have these values, it's only in the request

Reply:
Yes thanks, I'll remove it.